test/e2e/case36_empty_map_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+var _ = Describe("Test resource creation when there are empty labels in configurationPolicy", Ordered, func() {
```

Comment:
Ordered need?

Reply:
It looks like it isn't but since it's a cherry-pick, I'd rather not change it here.